spec/features/gym_search/show_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe 'gyms show page' do
+  before(:each) do
+    visit gym_path('BJBXzKYxQAXZKb5W6HrRnA')
+  end
+
+  it 'displays the gym and its information', :vcr do
+    expect(page).to have_content("Rishi's Community Yoga")
+    expect(page).to have_content('2750 S Wadsworth Blvd, Bldg D, Ste 202, Denver, CO 80227')
```

Comment:
I would suggest using `expect(page).to have_css `#name` type syntax, and not hard-coding the strings.  Unlikely given cassette's recorded against real data, but still does make for a more fragile test.  If we are getting back data that is rendered into div id's, I would consider that passing (and more dynamic) tests

Reply:
Added have_css to testing